README.md

```diff
@@ -100,17 +105,25 @@ Here are the steps to do so:
 1. Download the Oracle JDBC JAR files into `./oracle-libs/support` by
    following instructions in `./oracle-lib/README.md`. (The CMR must have these
    libraries to build but it does not depend on Oracle DB when running
-   locally. It uses a local in-memory database by default.)
+   locally. It uses a local in-memory database by default.) If you're reading this
+   guide on the web, [here is a handy link to the instructions.](https://github.com/nasa/Common-Metadata-Repository/tree/master/oracle-lib)
 1. With the JAR files downloaded to the proper location, you're now ready
    to install them for use by the CMR:" `cmr install oracle-libs`
 
-#### Building and Running CMR Dev System in a REPL
+#### Building and Running CMR Dev System in a REPL with CMR CLI tool
 
-1. `cmr setup profile` and then update the new `./dev-system/profiles.clj`
-   according to personal instructions provided by a CMR code developer.
-1. `cmr setup dev`
-1. `cmr start repl`
-1. Once given a Clojure prompt, run `(reset)`
+1. `cmr setup profile` and then update the new `./dev-system/profiles.clj` file.
+   it will look something like this:
+   ```
+   {:dev-config {:env {:cmr-metadata-db-password "<YOUR PASSWORD HERE>"
+                       :cmr-sys-dba-password "<YOUR PASSWORD HERE>"
+                       :cmr-bootstrap-password "<YOUR PASSWORD HERE>"
+                       :cmr-ingest-password "<YOUR PASSWORD HERE>"
+                       :cmr-rabbit-mq-password "<YOUR PASSWORD HERE>"}}}
```

Comment:
rabbit-mq is being purged in CMR-5319 since we don't use it anymore. Should probably remove this from dev-config.